### PR TITLE
feat: update county labels to show full name in listings list

### PR DESCRIPTION
**Pull request by ifranch**
# Pull Request Template

## Description

- Display the full and translated county name for listing cards

I tried also updating the labels for the landing and listing page search for the counties but it's not a straightforward change. The label is currently both used for display and for functionality (to see when a county is selected) so using the translations breaks the search functionality. The county search will need a rewrite and I would rather not tackle that during the bug bash.

## How Can This Be Tested/Reviewed?

- Go to the listings page on the public site
- Verify that the county name includes the word "County"
- Switch to another language and verify that the county name is translated

County names in English 
image

County names in Spanish
image

### CODE REVIEW

File: shared-helpers/src/locales/es.json [hunk 1]
```diff
@@ -406,7 +406,7 @@
   "counties.Solano": "Solano",
   "counties.Sonoma": "Sonoma",
   "counties.fullname.Alameda": "Condado de Alameda",
-  "counties.fullname.Contra Costa": "Condado de Contra Costa",
+  "counties.fullname.ContraCosta": "Condado de Contra Costa",
```
Comment (isaac-looker) [inline shared-helpers/src/locales/es.json:409]: Is removing the space a typo? https://github.com/metrotranscom/doorway/pull/225/files#diff-2adb536fd49c9a9de3b0305b9a8a4949a377e68e9ae1d6558c31c49f6068bb05R185 has a space.

**Review by isaac-looker — CHANGES_REQUESTED** [review]

File: shared-helpers/src/locales/es.json (hunk 1 above)
Comment (ifranch) [reply shared-helpers/src/locales/es.json:409]: This is intended. The message name in general.json is "counties.fullname.ContraCosta" so we have to use the same for all other languages. The name was not being properly translated, this is necessary.

**Review by isaac-looker — APPROVED** [review]

File: shared-helpers/src/locales/es.json (hunk 1 above)
Comment (phattya) [reply shared-helpers/src/locales/es.json:409]: So odd! I swore I already fixed this. Maybe I haven't pushed that PR yet. Great find @ifranch!!!

**Review by phattya — APPROVED** [review]
Wonderful! Thank you!

**Pull request merged**